parsl/providers/aws/aws.py

```diff
@@ -2,7 +2,12 @@
 import logging
 import os
 import time
+import io
 from string import Template
+from pathlib import Path
+
+from paramiko import SSHClient, AutoAddPolicy
+from scp import SCPClient
```

Comment:
I will move these dependencies such that they are not needed unless AWS is being used, like with `boto3` dep